### PR TITLE
Add an unique check for motion_category.prefix.

### PR DESCRIPTION
**Pull request by reiterl**
And add tests.

The first part of the linked issue.

### CODE REVIEW

File: openslides_backend/action/actions/motion_category/create_update_delete.py [hunk 1]
```diff
@@ -1,12 +1,48 @@
...
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        instance = super().update_instance(instance)
+        if instance.get("prefix"):
+            meeting_id = instance.get("meeting_id")
+            if meeting_id is None:
+                workflow = self.datastore.get(
+                    fqid_from_collection_and_id("motion_category", instance["id"]),
+                    ["meeting_id"],
+                )
+                meeting_id = workflow.get("meeting_id")
```
Comment (jsangmeister) [inline openslides_backend/action/actions/motion_category/create_update_delete.py:26]: Use the `get_meeting_id` method here

File: openslides_backend/action/actions/motion_category/create_update_delete.py [hunk 2]
```diff
@@ -1,12 +1,48 @@
...
+            if self.datastore.exists(
+                "motion_category",
+                And(
+                    FilterOperator("prefix", "=", instance["prefix"]),
+                    Not(FilterOperator("id", "=", instance["id"])),
```
Comment (jsangmeister) [inline openslides_backend/action/actions/motion_category/create_update_delete.py:31]:
```suggestion
                    FilterOperator("id", "!=", instance["id"]),
```

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/motion_category/create_update_delete.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/motion_category/create_update_delete.py]: Done.

File: openslides_backend/action/actions/motion_category/create_update_delete.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/motion_category/create_update_delete.py]: I now use `get_meeting_id().` Done.

**Review by r-peschke — APPROVED** [review]

**Pull request merged**